src/components/header/Header.tsx

```diff
@@ -44,7 +44,7 @@ export const Header = () => {
         </Styled.MenuItems>
       </Styled.Menu>
       <Styled.Toggle onClick={handleClick}>
-        {!isActive ? <MenuIcon /> : <CloseIcon />}
+        {!isActive ? <MenuIcon data-testid="hamburger" /> : <CloseIcon data-testid="close" />}
```

Comment:
small thing but I would call this `hamburger-icon` and `close-icon`

Reply:
Thanks, good suggestion! Will change this